src/JiraStatusNotifier/Domain/JiraConnector.php

```diff
@@ -46,18 +46,24 @@ public function handle(): array
     {
         $jiraBoard = new Board($this->input->getDaysForStatus());
         $company = Company::withNameAndProject($this->input->getCompanyName(), $this->input->getJiraProjectName());
-        $result = [];
 
         $ticketsByAssignee = (new TicketsByAssigneeClient(
             $this->jiraHttpClient,
             new JqlUrlFactory($jiraBoard, JqlUrlBuilder::inOpenSprints($company)),
             TicketFilter::notWithAssigneeKeys(...$this->input->getJiraUsersToIgnore())
         ))->fetchFromBoard($jiraBoard);
 
+        return $this->send($company, $ticketsByAssignee);
+    }
+
+    private function send(
+        Company $company,
+        TicketsByAssignee $ticketsByAssignee
+    ): array {
```

Comment:
I think you have forgotten to import the `TicketsByAssignee` class at the top of the file, no?
```php
use Chemaclass\JiraStatusNotifier\Domain\Jira\TicketsByAssignee\TicketsByAssignee;
```

Reply:
Also, in case the function has 1 or 2 arg, I usually prefer to have them in one line like this:
```php
private function send(Company $company, TicketsByAssignee $ticketsByAssignee): array
{
```

Reply:
Thanks for replying! Ya I forgot.. 😨